distributions/otelcol-contrib/otelcol-contrib.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
```

Comment:
I tested installing the RPM on a fresh Fedora 34 and verified that the systemd service was up and running.